.github/workflows/deploy-component-library.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy Component Library
+on:
+  push:
+    branches:
+      - 'main'
+jobs:
+  component-library-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.10'
```

Comment:
Any reason this one is 16.10 and SOM is 16.13?

Reply:
Just because 16.10 is what was already in the `.nvmrc` file for this repo. I just used the existing "known to work" version. We could probably update it, but it's not necessary for this PR